orangecontrib/text/widgets/owpubmed.py

```diff
@@ -0,0 +1,463 @@
+import os
+import re
+
+from PyQt4 import QtGui, QtCore
+from PyQt4.QtGui import *
+from validate_email import validate_email
+
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.widget import OWWidget
+from orangecontrib.text.corpus import Corpus
+from orangecontrib.text.pubmed import (
+    Pubmed, PUBMED_TEXT_FIELDS
+)
+
+
+def _i(name, icon_path='icons'):
+    widget_path = os.path.dirname(os.path.abspath(__file__))
+    return os.path.join(widget_path, icon_path, name)
+
+
+def eval_date(date):
+    if not date:
+        return True
+    pattern = re.compile('\d{4}/\d{2}/\d{2}')
+    return pattern.match(date)
+
+
+class Output:
+    CORPUS = 'Corpus'
+
+
+class OWPubmed(OWWidget):
+    name = 'Pubmed'
+    description = 'Load data from the Pubmed api.'
```

Comment:
Load data sounds strange when getting data from online API IMHO. Maybe fetch? Also loose the word `api`.